kafka-ui-react-app/src/components/Topics/Topic/Messages/Message.tsx

```diff
@@ -115,11 +118,13 @@ const Message: React.FC<Props> = ({
           <div>{formatTimestamp(timestamp)}</div>
         </td>
         <StyledDataCell title={key}>
+          <S.Icon>{keySerde === 'Fallback' && <WarningRedIcon />}</S.Icon>
```

Comment:
do we have an `enum` for this ? `'Fallback'`

Reply:
i didn't find